packages/app/src/common/components/AmountInput.tsx

```diff
@@ -18,6 +19,7 @@ interface AmountInputProps {
   setAmount: (amountStr: string) => void;
   setValid: (valid: boolean) => void;
   mode: UIMode;
```

Comment:
Didn't we want to get rid of it? Maybe we should actually rename the values to SEND and RECEIVE, where 0.1 is deducted on SEND

Reply:
Yeah, but I didn't know how.